<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlcoTestPageWidget</class>
 <widget class="QWidget" name="AlcoTestPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="top">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>124</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	
	background-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="photo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>124</y>
      <width>1280</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: rgb(255, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="topGradient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>124</y>
      <width>1280</width>
      <height>136</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(0, 0, 0), stop:1 rgba(0, 0, 0, 0));
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="stepFour">
    <property name="geometry">
     <rect>
      <x>904</x>
      <y>49</y>
      <width>216</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>04</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="stepTwo">
    <property name="geometry">
     <rect>
      <x>408</x>
      <y>49</y>
      <width>216</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>02</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="stepThree">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>49</y>
      <width>216</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 210, 51);
}</string>
    </property>
    <property name="text">
     <string>03 АЛКОТЕСТ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>26</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgba(0, 0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/logoSign.svg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="stepOne">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>49</y>
      <width>216</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>01</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
